src/app/helpers/serialization.tsx

```diff
@@ -114,9 +117,23 @@ const loadPlugins = (dir : string, rootSettings : any) => {
         component: componentPath
       }} as IInstallNeededPlugin
     } else {
-      return {...plugin, ...{
-        component: require(componentPath).default as React.ComponentType<any>
-      }} as IInstalledPlugin
+      try {
+        return {...plugin, ...{
```

Comment:
You can rewrite this to:

```javascript
return {
  ...plugin,
  component: require(componentPath).default as React.ComponentType<any>
} as IInstalledPlugin
```

Reply:
this is cool - i'm going to punt on this for now though, as there's quite a few places in the codebase that do this that deserve to be refactored at the same time. see https://github.com/bengreenier/overlayed/issues/36